sdks/python/apache_beam/transforms/external.py

```diff
@@ -655,11 +667,13 @@ class JavaJarExpansionService(object):
   argument which will spawn a subprocess using this jar to expand the
   transform.
   """
-  def __init__(self, path_to_jar, extra_args=None):
+  def __init__(self, path_to_jar, extra_args=None, classpath=None):
     if extra_args is None:
-      extra_args = ['{{PORT}}', f'--filesToStage={path_to_jar}']
+      to_stage = ','.join([path_to_jar] + list(classpath or []))
```

Comment:
Seems like we assume that the provided classpath only contains jars ? Classpath also supports directories and wild cards.
https://docs.oracle.com/javase/8/docs/technotes/tools/windows/classpath.html